doc/processing.md

```diff
@@ -0,0 +1,92 @@
+## How OAR Facilities Are Created And Matched
+
+The list of searchable facilities in the Open Apparel Registry is compiled from
+contributor submitted lists. The application uses statistical text analysis to
+compare newly submitted items to the existing list of facilities in an attempt
+to automatically match highly similar items and to suggest potential matches
+that require confirmation when there weaker similarities between items.
+
+Contributors submit their facility lists to the Open Apparel registry for
+processing by uploading lists in comma-separated value (CSV) format. All popular
+spreadsheet programs, including Microsoft Excel, can save a sheet in CSV format.
+The Open Apparel Registry provides a downloadable template, but any CSV can be
+read and processed if it meets the following criteria:
+
+  - Includes a header row with the following field names (in lower case)
+    - country
+    - name
+    - address
+  - Less than 5MB in size.
+
+Each line in a contributed facility list CSV file is processed separately and
+passes through three steps:
+
+  - _Parse_: The line is separated into distinct country, name, and address fields
+    and the country field is verified against a list of known countries.
+
+  - _Geocode_: The address and country are submitted to a service that returns a
+    location the globe corresponding to that country and address. This allows
+    the application to show the location of the facility on a map.
+
+  - _Match_: The country, name, and address are compared to the list of known
+    OAR facilities.
+    - If the country, name, and address are confidently matched to an existing
+      facility then that match is recorded. The facility details page will show a
+      link to contributor and show the submitted name and address as alternate
+      versions of the facility name and address.
+    - If one or more potential matches to an existing facility are found, but the
+      application is not confident enough to make the match automatically, the
+      contributor will be asked to "Accept" or "Reject" the match using buttons
+      within the application.
+    - If no potential matches to any existing facility are found, the application
+      creates a new facility using the country, name, and address and assigns a new
+      OAR ID number to the new facility.
+
+Contributed rows are matched to the existing facility list using statistical
+text analysis. The application calculates the probability that a contributed row
+is a match to an existing facility by comparing the country, name, and address
+fields individually and then joining those three comparisons together to create
+a final confidence score. The three individual comparisons are not given equal
+weight, however. The country of a submitted row, for example, must be an exact
+match.
```

Comment:
Would it be too much detail to explain how the name and address scores are weighted?

Reply:
I think getting into the details of the statistics is out of scope.